magprime/templates/static_views/super_swag_bag.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html>
+<head>
+  <title>Super Swag Bag</title>
+  <style type="text/css">
+    html, html * { font-family: Arial, Helvetica, sans-serif; }
+    img { max-width: 100%; }
+    h1 { text-align: center; }
+  </style>
+</head>
+<body>
+
+<h1>Super Swag Bag</h1>
+
+<p>
+  We've heard your feedback, and are making our supporter package swag bags
+  better than ever!
+</p>
+<p>
+  Are you tired of boring looking bags that are either empty, or filled with
+  stuff that is really boring. Well, boy, do we have a surprise for you. Our
```

Comment:
This first sentence should probably end in a question mark. It feels weird to have the question sentence construction ("Are you") without a question mark!